### PR TITLE
feat(implementation of carousel)! fix(filter and img)

### PR DESCRIPTION
**Pull request by DanielMino19**

### CODE REVIEW

File: src/components/catalog/ShopNavComponentNew.tsx [hunk 1]
```diff
@@ -86,7 +85,17 @@ export const ShopNavComponentNew = ({ handleFilterNav, updateFilteredData }:
...
     setSelectedFormulation(null);
+    setFilteredProducts([]);
   };
+ 
+  const allData = () => {
+    const allDbData = db;
+    if (selectedCategory){
+      updateFilteredData(allDbData);
+      resetFilters();
+      console.log(allDbData)
```
Comment (davidcarodsn) [inline src/components/catalog/ShopNavComponentNew.tsx]: Delete console.log

File: src/components/catalog/ShopNavComponentNew.tsx [hunk 2]
```diff
@@ -86,7 +85,17 @@ export const ShopNavComponentNew = ({ handleFilterNav, updateFilteredData }:
   const resetFilters = () => {
     setSelectedCategory(null);
     setSelectedFormulation(null);
+    setFilteredProducts([]);
   };
+ 
+  const allData = () => {
+    const allDbData = db;
```
Comment (davidcarodsn) [inline src/components/catalog/ShopNavComponentNew.tsx]: You can delete constant for allData, instead you can set db as a function prop

File: src/components/catalog/ShopComponent.tsx [hunk 3]
```diff
@@ -28,12 +31,22 @@ const ShopComponent:FC<ShopComponentProps> = ({ filter }) => {
   const updateFilteredData = (filteredData: ProductData[]) => {
     setProductData(filteredData);
   };
-  
+
+
+  const getCurrentPagePosts = () => {
```
Comment (davidcarodsn) [inline src/components/catalog/ShopComponent.tsx:36]: You can move this function on utils/helpers

Comment (DanielMino19) [reply src/components/catalog/ShopComponent.tsx:36]: I don't want to

Comment (davidcarodsn) [reply src/components/catalog/ShopComponent.tsx:36]: Si te volves a rebelar vas a terminar en una bolqueta en medio de la 9 de Julio

Comment (DanielMino19) [reply src/components/catalog/ShopComponent.tsx:36]: primero tenes que encontrarme

**Comment by davidcarodsn** [comment]
LGTM

**Pull request merged**